_im-plugin/append-only-index.md

```diff
@@ -0,0 +1,42 @@
+---
+layout: default
+title: Append-only Index
+nav_order: 14
+redirect_from:
+  - /opensearch/append-only-index/
+---
+
+# Append only index
+
+An append-only index is an immutable index that only allows document ingestion (appending) while blocking all forms of updates or deletions after initial document creation.
+When you enable the append-only setting for an index, OpenSearch prevents any modifications to existing documents. You can only add new documents to the index.
+
+When an index is configured as append-only, the following operations will return an error:
+- Document update call (Update API)
+- Document delete call (Delete API)
+- Update by query call
+- Delete by query call
+- Bulk API call that include update, delete, or upsert actions
```

Comment:
Its not the entire api which throws the error, right? only the documents with the denied operations? Can we reword to reflect the same?

Reply:
Addressed the comments.